File: src/main/java/de/sean/splugin/spigot/commands/LockExecutor.java
package de.sean.splugin.spigot.commands;

/* SPlugin */
import de.sean.splugin.SPlugin;
import de.sean.splugin.util.SLockUtil;

/* Java */
import org.bukkit.configuration.file.FileConfiguration;
import org.jetbrains.annotations.NotNull;

/* Spigot */
import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class LockExecutor implements CommandExecutor {
    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        final Player player = Bukkit.getPlayer(sender.getName());
        if (player == null) return false;
        switch (args[0]) {
            case "place":
                final FileConfiguration config = SPlugin.instance.getConfig();
                config.set("Players." + player.getUniqueId() + ".NotLockOnPlace", !config.getBoolean("players." + player.getUniqueId() + ".notLockOnPlace"));
                SPlugin.instance.saveConfig();
                break;
            default:
                return false;
        }
        return false;
    }
}

File: src/main/kotlin/de/sean/splugin/spigot/commands/LockExecutor.kt
package de.sean.splugin.spigot.commands

import de.sean.splugin.SPlugin
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class LockExecutor : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        val player = Bukkit.getPlayer(sender.name) ?: return false
        when (args[0]) {
            "place" -> {
                val config = SPlugin.instance.config
                config["Players." + player.uniqueId + ".NotLockOnPlace"] = !config.getBoolean("players." + player.uniqueId + ".notLockOnPlace")
                SPlugin.instance.saveConfig()
            }
            else -> return false
        }
        return false
    }
}
